### DISCUSSION TITLE
Should the version of lsp4ij installed and the version of it in the plugin the same?

### DISCUSSION THREAD

**Discussion by onriv** [redhat-developer/lsp4ij#458] [Q&A] [answered]
Hi I got lsp4ij in a plugin via the way https://github.com/onriv/lean4ij/blob/main/build.gradle.kts#L90 adding the lastest nightly version  and in the Intellij Idea I installed lsp4ij 3.0. Should I keep them the same? Or is lsp4ij bundled in the plugin too?
It seems to be some dependencies issue for me that is quite strange to check. The pipeline shows I got `0.4.0-20240814-141029@nightly` https://github.com/onriv/lean4ij/actions/runs/10392273209/job/28777068417#step:7:17 but after installing it, the idea.log file shows I am depending on a version not existing.

**Comment by onriv** [comment]
A new try seems working now. Though the installed lsp4ij plugin is with version 0.3.0 and in plugin.xml of I depends on some nightly build

**Reply by angelozerr** [reply, marked as answer]
I don't know if I answer to your question, but your plugin doesn't embed LSP4IJ library since LSP4IJ is a plugin. It is a great news for LSP4IJ that you create a new plugin based on LSP4IJ.

Could you please create a PR to add your plugin in https://github.com/redhat-developer/lsp4ij?tab=readme-ov-file#who-is-using-lsp4ij please. And if you have time, don't hesitate to add a review in https://plugins.jetbrains.com/plugin/23257-lsp4ij/edit/reviews to help us to promote LSP4IJ. Thanks!

**Reply by onriv** [reply]
Thanks! Yeah sure I create [a PR](https://github.com/redhat-developer/lsp4ij/pull/460) for updating the readme and will add a review in the marketplace later!

**Comment by angelozerr** [comment]
@onriv can we close this discussion?

**Comment by onriv** [comment]
Yeah sure I close it now. Thanks
